pom.xml

```diff
@@ -45,14 +45,14 @@
       Kafka 0.10 API bug:
       Cannot auto-commit offsets for group ... since the coordinator is unknown
     -->
-    <kafka.version>0.9.0.1</kafka.version>
+    <kafka.version>0.11.0.1</kafka.version>
```

Comment:
I suspect you may get some pushback here as the original author was doing work with hdp. Afaik there is no hdp release which includes 0.11. Nothing to change now just something to keep in mind